recipes/simbody/all/conanfile.py

```diff
@@ -0,0 +1,92 @@
+from conan import ConanFile, tools
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.scm import Version
+from conan.tools.files import copy, get, rmdir, save
+import os
+import textwrap
+
+required_conan_version = ">=1.50.0"
+
+class SimbodyConan(ConanFile):
+    name = "simbody"
+    description = "High-performance, open-source toolkit for science- and engineering-quality simulation"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/simbody/simbody"
+    topics = ("high-performance", "science", "simulation")
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = "CMakeLists.txt"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+
+    def validate(self):
+        if not self.dependencies["openblas"].options.build_lapack:
+            raise ConanInvalidConfiguration(f"{self.name} requires the openblas:build_lapack: True")
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    @property
+    def _module_file_rel_path(self):
+        return os.path.join("lib", "cmake", f"conan-official-{self.name}-variables.cmake")
+
+    def requirements(self):
+        self.requires("openblas/0.3.17")
+        self.requires("opengl/system")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version],  
+                    destination=self.source_folder, strip_root=True)
+
+    def layout(self):
+        cmake_layout(self, src_folder="Simbody/src")
+
+    def generate(self):
+        tc = CMakeDeps(self)
+        tc.generate()
+
+    def build(self):
+        cmake = CMake(self)
```

Comment:
```suggestion
        apply_conandata_patches(self)
        cmake = CMake(self)
```
Don't forget to apply patches.

Reply:
done